tests/codegen/expected_bundles/local_bundle_prefix_suffix.yml

```diff
@@ -8,6 +8,7 @@ environments:
           email_notifications: null
           notification_settings: null
           webhook_notifications: null
+          health: null
```

Comment:
I know for may of our tests, we provided null values to validate if the key's are present. Can you please provide some value for health in the workflow that generates/validates these files. In the subsequent pr's, we will add values for notifications etc as well.